src/vsr/superblock.zig

```diff
@@ -207,7 +207,8 @@ pub const SuperBlockSector = extern struct {
         assert(superblock.version == SuperBlockVersion);
         assert(superblock.flags == 0);
 
-        for (mem.bytesAsSlice(u64, &superblock.reserved)) |word| assert(word == 0);
+        assert(@bitCast(u32, superblock.reserved[0..4].*) == 0);
+        for (mem.bytesAsSlice(u64, superblock.reserved[4..])) |word| assert(word == 0);
```

Comment:
@ifreund Our of curiosity, why split the assertion up like this?

Reply:
the new length of the reserved array is not evenly divisible by 8 byte words, so we can't use `mem.byteAsSlice(u64, ...` on the array as a whole. 

It may be the case that LLVM has no problem generating the code we want here that compares 8 byte words in the hot loop instead of single bytes at a time, but I haven't checked and the code you see here is my attempt to keep the style as close to the status quo as possible.

Reply:
:+1: Got it, thanks!